lib/utils.coffee

```diff
@@ -27,11 +27,11 @@ THE SOFTWARE.
 # @function
 # @protected
 #
-# @param {String} uuid - device uuid
+# @param {Object} device - device
 # @returns {String} logs channel
 #
 # @example
 # channel = utils.getChannel('...')
 ###
-exports.getChannel = (uuid) ->
-	return "device-#{uuid}-logs"
+exports.getChannel = (device) ->
+	return device.logs_channel or "device-#{device.uuid}-logs"
```

Comment:
I don't think this change is correct 

https://bitbucket.org/rulemotion/resin-ui/pull-requests/124/if-available-use-devicelogs_channel-for/diff#Lsrc/scripts/modules/device-logs/directives/device-logs-directive.coffeeT18
https://bitbucket.org/rulemotion/resin-ui/pull-requests/124/if-available-use-devicelogs_channel-for/diff#Lsrc/scripts/modules/device-logs/services/device-logs-service.coffeeT35